tools/extra/fpgadiag/fpgastats.py

```diff
@@ -127,7 +127,8 @@ def print_device_stats(self, handler, stats, reg, length, fifo=False):
                 v = self.fpga_eth_reg_read(handler, comp, dev, addr)
                 data += (0xffffffff & v) << (32 * l)
             print("{0: <12}".format(data), end=' | ')
-
+        # Output prints in new line
+        print()
```

Comment:
Why are we doing this?

Reply:
Somehow adding this print helps format the output in rows and columns. Without it, it's difficult to understand.   
![image](https://user-images.githubusercontent.com/42047493/58058561-4a761a00-7b1e-11e9-880f-b981d6ccf8ca.png)

Reply:
for output formatting